lib/text.rb

```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+
+class Text
+
+
+  def send_text
+    order_time = (Time.new + 3600).strftime("%H:%M")
+    account_sid = "AC0fec486c68cb4eb5041930075e919693"
+    auth_token = "5f688760630ddbb8e40ab6244c303b77"
+    @client = Twilio::REST::Client.new account_sid, auth_token
+
+    sms =  @client.account.messages.create(
+      :from => +441732601037,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax.<br>Style/NumericLiterals: Separate every 3 digits in the integer portion of a number with underscores(_).